src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <div class="col-sm-12 col-lg-12">
+      <div>
+          <cd-table-actions class="btn-group"
+              [permission]="permission"
+              [selection]="selection"
+              [tableActions]="createTableActions">
+          </cd-table-actions>
+      </div><br>
```

Comment:
instead of using `br` you should use[ `mt-1` or `mb-1`](https://getbootstrap.com/docs/4.0/utilities/spacing/) depending on where you are putting the margin. Its more consistent across our different brandings. (IBM, RHCS..)